src/mapOfPeople.js

```diff
@@ -40,7 +40,19 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const team = new Map();
+  let count = 0;
+
+  people.forEach(number => {
```

Comment:
```suggestion
  people.forEach((number, index) => {
```
use index instead of count